resources/.eslintrc-standards--test-override.js

```diff
@@ -1,11 +1,21 @@
 module.exports = {
     "rules": {
-        "max-nested-callbacks":     "off",     // our tests are deep like our love
-        "newline-per-chained-call": "off",     // keep our tests a bit more compact
-        "no-invalid-this":          "off",     // the context of `this` inside mocha functions is fine. Go away.
-        "no-magic-numbers":         "off",     // too fussy for test set up
-        "no-unused-expressions":    "off",     // allows expect(true).to.be.true;
-        "quote-props":              "off",     // ignore quotes on props, fairly common for JSON responses in tests ok!
-        "object-curly-spacing":     "off"      // ignore object spacing, again typically JSON has spaces. We do not. But it's okay!
+        "max-nested-callbacks":     "off",      // our tests are deep like our love
+        "newline-per-chained-call": "off",      // keep our tests a bit more compact
+        "no-invalid-this":          "off",      // the context of `this` inside mocha functions is fine. Go away.
+        "no-magic-numbers":         "off",      // too fussy for test set up
+        "no-unused-expressions":    "off",      // allows expect(true).to.be.true;
+        "no-restricted-properties": ["error", { // disallow certain properties on certain objects
+            "object": "describe",
+            "property": "only"
+        }, {
+            "object": "context",
+            "property": "only"
+        }, {
+            "object": "it",
+            "property": "only"
+        }],
```

Comment:
yep sure could. good idea, I'll do that. 

(though it would be defined in the base rules, not these test specific ones)